.github/workflows/release.yaml

```diff
@@ -334,3 +341,60 @@ jobs:
             channel_id: ${{ inputs.slack-channel-id }}
             title: '[${{ steps.component-descriptor.outputs.component-version }} released]'
             file: /tmp/release-notes/slack-release-notes.md
+
+      - name: reset-repository-to-release-commit
```

Comment:
we _could_ also "import" the release-commit, as done in release-action (which would save us a git-fetch-operation / work even if release-commit would not be tagged); I would deem this to be preferable (-> less dependencies against remote repository).